yarn.lock

```diff
@@ -7290,6 +7290,13 @@ axios@0.21.4, axios@^0.21.1, axios@^0.21.2:
   dependencies:
     follow-redirects "^1.14.0"
 
+axios@0.24.0:
```

Comment:
This does not look right 🤔

Reply:
I'm assuming one of the dependencies now requires Axios, so might be fine to keep 🤔